content/docs/1.6.0/high-availability/recover-volume-when-read-only.md

```diff
@@ -0,0 +1,11 @@
+---
+  title: Recover Volume When It Becomes Read Only
+  weight: 1
+---
+
+The state of a volume can change to read-only when IO errors occur. IO errors can be caused by network disconnection, high disk latency, abnormal node shutdown, and other issues.
```

Comment:
Suggest briefly describing the cases. 

network disconnection: between engine and replicas
high-disk latency: latency between replica and the corresponding disk
abnormal node shutdown: node down where the engine is running.

Reply:
Fixed.